web/ui/dashboard/src/app/private/pages/project/events/event-deliveries/event-deliveries.component.ts

```diff
@@ -102,6 +102,7 @@ export class EventDeliveriesComponent implements OnInit {
 			const eventDeliveriesResponse = await this.eventDeliveriesRequest({ pageNo: page, eventId: this.eventDeliveryFilteredByEventId, startDate, endDate });
 			this.eventDeliveries = eventDeliveriesResponse.data;
 			this.displayedEventDeliveries = this.generalService.setContentDisplayed(eventDeliveriesResponse.data.content);
+			this.displayedEventDeliveries = this.displayedEventDeliveries?.sort((a, b) => new Date(b.date).getDate() - new Date(a.date).getDate());
```

Comment:
Why not let this happen in the `setContentDisplayed` method @Oluwadaminiola. So everything from there will be arranged.

Reply:
done @horlah